HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -9576,6 +9722,447 @@ def ev_charger
+        fail "One or more referenced components '#{@component_idrefs.join("', '")}' not valid for service feeder '#{@id}'" if ![HPXML::ElectricPanelLoadTypeClothesDryer].include?(@type)
+      end
+      if !dishwashers.empty?
+        fail "One or more referenced components '#{@component_idrefs.join("', '")}' not valid for service feeder '#{@id}'" if ![HPXML::ElectricPanelLoadTypeDishwasher].include?(@type)
+      end
+      if !cooking_ranges.empty?
+        fail "One or more referenced components '#{@component_idrefs.join("', '")}' not valid for service feeder '#{@id}'" if ![HPXML::ElectricPanelLoadTypeRangeOven].include?(@type)
+      end
+      if !ventilation_fans.empty?
+        fail "One or more referenced components '#{@component_idrefs.join("', '")}' not valid for service feeder '#{@id}'" if ![HPXML::ElectricPanelLoadTypeMechVent].include?(@type)
+      end
+      if !permanent_spa_pumps.empty?
+        fail "One or more referenced components '#{@component_idrefs.join("', '")}' not valid for service feeder '#{@id}'" if ![HPXML::ElectricPanelLoadTypePermanentSpaPump].include?(@type)
+      end
+      if !permanent_spa_heaters.empty?
+        fail "One or more referenced components '#{@component_idrefs.join("', '")}' not valid for service feeder '#{@id}'" if ![HPXML::ElectricPanelLoadTypePermanentSpaHeater].include?(@type)
+      end
+      if !pool_pumps.empty?
+        fail "One or more referenced components '#{@component_idrefs.join("', '")}' not valid for service feeder '#{@id}'" if ![HPXML::ElectricPanelLoadTypePoolPump].include?(@type)
+      end
+      if !pool_heaters.empty?
+        fail "One or more referenced components '#{@component_idrefs.join("', '")}' not valid for service feeder '#{@id}'" if ![HPXML::ElectricPanelLoadTypePoolHeater].include?(@type)
+      end
+      if !plug_load_well_pumps.empty?
+        fail "One or more referenced components '#{@component_idrefs.join("', '")}' not valid for service feeder '#{@id}'" if ![HPXML::ElectricPanelLoadTypeWellPump].include?(@type)
+      end
+      if !plug_load_vehicles.empty?
+        fail "One or more referenced components '#{@component_idrefs.join("', '")}' not valid for service feeder '#{@id}'" if ![HPXML::ElectricPanelLoadTypeElectricVehicleCharging].include?(@type)
+      end
+      if !ev_chargers.empty?
+        fail "One or more referenced components '#{@component_idrefs.join("', '")}' not valid for service feeder '#{@id}'" if ![HPXML::ElectricPanelLoadTypeElectricVehicleCharging].include?(@type)
+      end
+
+      list = heating_systems + cooling_systems + heat_pumps + water_heating_systems + clothes_dryers + dishwashers + cooking_ranges + ventilation_fans + permanent_spa_pumps + permanent_spa_heaters + pool_pumps + pool_heaters + plug_load_well_pumps + plug_load_vehicles + ev_chargers
+      if @component_idrefs.size > list.size
+        fail "One or more referenced components '#{@component_idrefs.join("', '")}' not found for service feeder '#{@id}'."
```

Comment:
I'm confused. Why did we get rid of this error check for BranchCircuits but it's still here for ServiceFeeders?